### ISSUE TITLE
3 Pre-release 2

### ISSUE THREAD

**Issue by Ambrevar** [atlas-engineer/nyxt#2565]
I'll release 3 pre-release 2 Very Soon©.  Any last minute wish?

On my end:
- [x] https://github.com/atlas-engineer/nyxt/issues/2535
  See https://github.com/atlas-engineer/nyxt/pull/2575.
- [x] https://github.com/atlas-engineer/nyxt/issues/2449
- [x] https://github.com/atlas-engineer/nyxt/commit/fd3df52a1c26608701c7f0098e2f9c86e2b96486

Also waiting on:
- [x] https://github.com/atlas-engineer/nyxt/pull/2574

@aartaka @aadcg @jmercouris

**Comment by aadcg** [comment]
I think #2555 will probably be merged soon.

**Comment by jmercouris** [comment]
No last minute wishes, I just wish for happiness of everyone on the team :-)

**Comment by Ambrevar** [comment]
@aadcg I say let's postpone it for after pre-release 2, as this is a rather significant change with mid-potential for bugs :)

**Comment by aartaka** [comment]
https://github.com/atlas-engineer/nyxt/commit/fd3df52a1c26608701c7f0098e2f9c86e2b96486 I'd do, but I guess I'm more needed in completing #2238 :D

**Comment by aadcg** [comment]
Fair enough @Ambrevar.

**Comment by Ambrevar** [comment]
@aartaka Have a look at my attempt in https://github.com/atlas-engineer/nyxt/pull/2566.

**Comment by hendursaga** [comment]
I'd say go for it Soon (TM) - perhaps you'd want to pin this issue 'til it's closed?

**Comment by Ambrevar** [comment]
Waiting for 2 pull requests (see original post) and we are good to go!

Please all review the Change Log and migration guide.

**Comment by aadcg** [comment]
Reviewing the changelog.

**Comment by Ambrevar** [comment]
I guess we are good to go for a release on Monday when #2578 is merged.

**Comment by danrobi11** [comment]
Hello. The 3-pre-release-2 release note link does not work. 

- https://nyxt.atlas.engineer/article/release-3-pre-release-2.org

**Comment by aartaka** [comment]
Indeed, it doesn't... yet!

CC @jmercouris ;)

**Comment by jmercouris** [comment]
It is done!
